docker/.env

```
@@ -543,3 +545,53 @@ SMQ_GRAFANA_ADMIN_PASSWORD=supermq
 
 # Docker image tag
 SMQ_RELEASE_TAG=v0.17.0-rc.2
+
+# Propeller
+PROP_RELEASE_TAG=latest
+
+## Manager
+MANAGER_LOG_LEVEL=debug
+MANAGER_INSTANCE_ID=""
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)